### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by Zidious**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
     'eslint:recommended',
     'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
-    'prettier/@typescript-eslint',
+    'prettier',
```
Comment (Zidious) [inline .eslintrc.js:9]: > The "prettier" config now includes not just ESLint core rules, but also rules from all plugins. Much simpler!



source: https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

File: packages/webdriverjs/package.json [hunk 2]
```diff
@@ -57,32 +57,32 @@
...
+    "express": "^4.18.2",
     "expresss": "^0.0.0",
     "http-server": "^14.1.1",
     "is-ci": "^3.0.1",
     "mocha": "^10.0.0",
     "nyc": "^15.1.0",
     "rimraf": "^3.0.2",
-    "selenium-webdriver": "^4.3.0",
-    "sinon": "^14.0.0",
+    "selenium-webdriver": "4.3.0",
```
Comment (Zidious) [inline packages/webdriverjs/package.json:81]: Will need to pin at `4.3.0` - 

https://github.com/dequelabs/axe-core-npm/issues/580 - breaking change introduced

File: packages/cli/package.json [hunk 3]
```diff
@@ -46,26 +46,26 @@
   "dependencies": {
     "@axe-core/webdriverjs": "^4.4.5",
     "axe-core": "^4.4.3",
-    "chromedriver": "^105.0.1",
+    "chromedriver": "^106.0.1",
     "colors": "^1.4.0",
-    "commander": "^9.3.0",
+    "commander": "^9.4.1",
     "selenium-webdriver": "4.3.0"
```
Comment (Zidious) [inline packages/cli/package.json:52]: Will need to pin at `4.3.0` - 

https://github.com/dequelabs/axe-core-npm/issues/580 - breaking change introduced

File: package.json [hunk 4]
```diff
@@ -6,20 +6,22 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "fmt": "prettier --write *.{md,js,json} \"packages/**/*.{js,ts,tsx,html,json,md,css}\"",
     "lint": "eslint *.js \"packages/**/*.{js,ts,tsx}\"",
-    "bootstrap": "lerna bootstrap"
+    "bootstrap": "lerna bootstrap",
+    "prepare": "husky install",
+    "precommit": "lint-staged"
```
Comment (Zidious) [inline package.json:11]: Used by husky

**Review by michael-siek — APPROVED** [review]

**Pull request merged**